examples/user-management/nextjs-user-management/app/account/account-form.tsx

```diff
@@ -4,136 +4,139 @@ import { createClient } from '@/utils/supabase/client'
 import { type User } from '@supabase/supabase-js'
```

Comment:
Double check the line numbers for this include, the changes have shifted the lines and the code inclusion is off:

<img width="961" height="611" alt="Image" src="https://github.com/user-attachments/assets/611708da-9d5b-4b46-8170-1bafe07b1659" />

Reply:
There's a few other oddities with this I noticed and will fix up, some of it from trying the inclusion, some from older versions of the doc.